app/views/import_wizards/index.haml

```diff
@@ -1,5 +1,13 @@
 :javascript
-  $(function() { $('#upload').change(function() { $('#upload_form').submit() }); });
+  $(function() {
+    $('#upload').change(function() { $('#upload_form').submit() });
+    $(".fexport").on("click", function(){
+      document.getElementById('upload').click();
+    });
+    $(".syes_no").on("click", function(){
+      $('#upload_form_spreadsheet').submit();
+    });
```

Comment:
Can we add IDs to these clickable elements? `.fexport` is referring to a CSS class, which may be applied to multiple elements - but we probably don't want _other_ `.fexport`s to trigger this very same action.